lib/ModernTribe/ruleset.xml

```diff
@@ -59,6 +59,13 @@
            functions, methods, if, elseif, foreach, for, and switch blocks. -->
     <rule ref="WordPress.WhiteSpace.ControlStructureSpacing"/>
 
+    <!-- No short echos allowed. WordPress-VIP-GO overides the default 
+        severity.  -->
+    <rule ref="Generic.PHP.DisallowShortOpenTag.EchoFound">
```

Comment:
Actually, should move this to the proper `Generic` section below.

Reply:
Should it?  I was thinking that it should be as close to the WordPress Standards that are removing it in the first place, so if we pull that standard, it is easy to see that you can also pull this.

Reply:
Definitely, all of these rules are grouped by namespace, this would be the only rule in the set that wouldn't be. It's not a `WordPress` namespace, even if their rule set provides an original definition for it.

Reply:
Thanks, should be good now.